services/static-webserver/client/source/class/osparc/NewRelease.js

```diff
@@ -26,9 +26,37 @@ qx.Class.define("osparc.NewRelease", {
     this.__buildLayout();
   },
 
+  statics: {
+    /**
+     * Compare the version logged in the cache with the one being shown
+     */
+    firstTimeISeeThisFrontend: function() {
+      let isIt = false;
+      const lastUICommit = osparc.utils.Utils.localCache.getLastCommitVcsRefUI();
+      const thisUICommit = osparc.utils.LibVersions.getVcsRefUI();
+      if (lastUICommit && thisUICommit) {
+        isIt = lastUICommit !== thisUICommit;
```

Comment:
Is it? ;) Why not simply return as in you do in the other function

Reply:
Because I want to save in the cache the commit id right after this line...